aniso8601/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
-  name: sphinx_rtd_theme
-  version: "0.1.8"
+  name: aniso8601
```

Comment:
Did you really mean to replace the sphinx_rtd_theme package here?  I don't think aniso is a replacement.